pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -24,21 +24,15 @@ import (
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/route"
 	http_conn "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
 	"github.com/gogo/protobuf/types"
```

Comment:
If move `types.BoolValue` into pkg/proto/types.go, then this imported package is not needed any more.

Please fix https://github.com/istio/istio/pull/9216/files#diff-9eec7bb8f46726e51ab373e7b8e11ca7L428

Reply:
done